app/profile/userSchema.ts

```
@@ -0,0 +1,45 @@
+import { z } from 'zod'
+
+export const userSchema = z.object({
+    address: z.string().optional().nullable(),
+    badges: z.any().optional().nullable(), // Json type
+    banned: z.boolean().optional().nullable(),
+    bio: z.string().optional().nullable(),
+    birthday: z.string().optional().nullable().refine((date) => !date || !isNaN(Date.parse(date)), {
```

Biome

[error] isNaN is unsafe. It attempts a type coercion. Use Number.isNaN instead.

See the MDN documentation for more details.
Unsafe fix: Use Number.isNaN instead.


(lint/suspicious/noGlobalIsNan)